bin/alluxio-monitor.sh

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+LAUNCHER=
+# If debugging is enabled propagate that through to sub-shells
+if [[ "$-" == *x* ]]; then
+  LAUNCHER="bash -x"
+fi
+BIN=$(cd "$( dirname "$( readlink "$0" || echo "$0" )" )"; pwd)
+
+USAGE="Usage: alluxio-monitor.sh [-hL] ACTION [host1,host2,...]
+Where ACTION is one of:
+  all                \tStart monitors for all masters, proxies, and workers nodes.
+  local              \tStart monitors for all process locally.
+  master             \tStart a master monitor on this node.
+  masters            \tStart monitors for all masters nodes.
+  worker             \tStart a worker monitor on this node.
+  workers            \tStart monitors for all workers nodes.
+  proxy              \tStart the proxy monitor on this node.
+  proxies            \tStart monitors for all proxies nodes.
+
+-L  enables the log mode, this option disable the monitor checks and causes alluxio-monitor to only print the node log tail.
+-h  display this help.
+"
+
+RED='\033[1;31m'
+GREEN='\033[1;32m'
+PURPLE='\033[0;35m'
+CYAN='\033[1;36m'
+NC='\033[0m'
+BOLD='\e[1m'
```

Comment:
I really like the idea to include styling, it makes the tool very pleasant to use. Could we use `tput` to define these? When I use `iTerm2` + zshell, the bold doesn't come through and I get
```
\e[1m-----------------------------------------\e[21m
Starting to monitor the master service.
\e[1m-----------------------------------------\e[21m
```
https://linux.101hacks.com/ps1-examples/prompt-color-using-tput/

Reply:
I see, I just added the bold, didn't know that have that effect in other shells, I will update this, thanks!

Reply:
Done